exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,8 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
-
+function containsCode(sentence) {
+  if (sentence.startsWith("code")){
```

Comment:
startsWith is not the correct method here. The function should check that ```sentence``` has the string "code" in it but not neccesarily at the start.